lib/metasploit/framework/login_scanner/http.rb

```diff
@@ -260,14 +276,20 @@ def send_request(opts)
 
           begin
             cli.connect
-            req = cli.request_cgi(opts)
+            # Send CGI compatible request by default
+            req = if opts['requestcgi'].nil? || opts['requestcgi']
+                    cli.request_cgi(opts)
+                  else
+                    cli.request_raw(opts)
+                  end
+
```

Comment:
I think for consistency, this should follow the existing approach:

```suggestion
            req = opts[:cgi] ? cli.request_cgi(opts) : cli.request_raw(opts)
```

As defined by:

https://github.com/rapid7/metasploit-framework/blob/f4745f884a835ffc07b9dd1158ac70983315dfdc/lib/msf/core/exploit/remote/http_client.rb#L433